src/Text/Pandoc/Writers/Jira.hs

```diff
@@ -193,6 +193,7 @@ toJiraInlines inlines = do
         Code _ cs          -> return . singleton $
                               Jira.Monospaced (escapeSpecialChars cs)
         Emph xs            -> styled Jira.Emphasis xs
+        Underline xs       -> styled Jira.Insert xs
```

Comment:
Semantically Insert seems different than underline, even if underline is used to indicate it.  But this is probably okay for now.  (At some point we might add Inserted and Deleted elements.)